coala

```diff
@@ -15,15 +15,14 @@
 
 import sys
 
-# Start ignoring PyImportSortBear, we need to assert the version before
-# importing things that could cause an ImportError
+# Start ignoring PyImportSortBear and PycodestyleBear*, we need to assert the
+# version before importing things that could cause an ImportError
 from coalib import assert_supported_version
 
 assert_supported_version()
-# Stop ignoring
 
```

Comment:
It has to stop ignoring after the line importing the module main from coalib.coala. Otherwise, PycodestyleBear rises error E402 because the function assert_supported_version() is called before the import.